modules/virt-custom-kubemacpool-range.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-using-mac-address-pool-for-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-custom-kubemacpool-range_{context}"]
+= Customizing the MAC pool range
+
+[role="_abstract"]
+KubeMacPool works by allocating MAC addresses to VMs from a range. The `rangeStart` and `rangeEnd` parameters in the `HyperConverged` custom resource (CR) define the MAC pool range.
```

Comment:
In the `HyperConverged`?
@RamLavi Isn't the KMP range defined in the KMP MAC range config-map?

Reply:
the feature is about allowing the user to configure the KMP range via HCO CR. It eventually gets reconciled to the KMP range configmap